hs/DrawResponse.hs

```diff
@@ -31,15 +29,13 @@ drawResponse =
 drawHtml :: H.Html
 drawHtml = H.div ! A.id "aboutDiv" $ do
   H.h1 "Draw a Graph"
-  H.p "Here you can create a graph yourself!"
 
-button :: H.Html
-button = createTag H.div "target" "" $ do "Click Here"
-
---modePanel :: H.Html
---modePanel = createTag H.div "mode-select" "col-md-2 col-xs-6" $ do "HAHA"
-
--- <canvas id="myCanvas" width="200" height="100"></canvas>
-theCanvas :: H.Html
-theCanvas = H.canvas ! A.id "theCanvas" ! A.width "800" ! A.height"375" $ do "No canvas support."
+modePanel :: H.Html
+modePanel = createTag H.div "mode-panel" "" $ do 
+  createTag H.div "node-mode" "mode" $ do "NODE"
```

Comment:
You can omit the `$ do` for these, and just pass in `"NODE"` as an argument to `createTag`.